regolith/helpers/a_presentationhelper.py

```diff
@@ -66,7 +66,8 @@ def subparser(subpi):
     subpi.add_argument("-n", "--notes", nargs="+",
                        help="note or notes to be inserted as a list into the notes field, "
                             "separate notes with spaces.  Place inside quotes if the note "
-                            "itself contains spaces."
+                            "itself contains spaces.",
+                       default=[]
```

Comment:
I think more readable is to use `default = None` or sthg like that, then in the code have an
```
if rc.notes is not None:
    do something
```
which will make your intent clearer (with the same outcome)

Reply:
Sounds great. I will make these changes.